<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnergySourceDialog</class>
 <widget class="QDialog" name="EnergySourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>269</width>
    <height>295</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Assign share HEATING and DHW</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/wizard.png</normaloff>:/images/wizard.png</iconset>
  </property>
  <widget class="QLabel" name="labelDHW">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>% DHW (0,100)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxDHW">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000100000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="labelHEATING">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>% HEATING (0,100)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxHEATING">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="maximum">
    <double>100.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.000100000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfirmParams">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	border: 1px solid black;
	border-radius: 5px;
	background-color: white;
	
}</string>
   </property>
   <property name="text">
    <string>Enter the share percentages for HEATING and DHW:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
 </resources>
 <connections/>
</ui>
